### PR TITLE
removed libusb and librtlsdr

### PR DESCRIPTION
**Pull request by BMillikan**

### CODE REVIEW

**Comment by BMillikan** [comment]
removed libusb and rtlsdr libraries

File: .gitmodules [hunk 1]
```diff
@@ -4,9 +4,9 @@
...
-[submodule "libusb"]
-	path = libusb
-	url = https://github.com/ankurvdev/libusb
-[submodule "rtl-sdr"]
-	path = rtl-sdr
-	url = https://github.com/osmocom/rtl-sdr
+[submodule "libusb-1.0"]
+	path = libusb-1.0
+	url = https://github.com/libusb/libusb.git
+[submodule "librtlsdr"]
```
Comment (ankurvdev) [inline .gitmodules:10]: This seems to be older and less actively maintained than the osmocom one which is the official version.

Any reason for the change ?

**Comment by BMillikan** [comment]
Sorry about that.  I’m a fit noob.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by BMillikan** [comment]
Git

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by BMillikan** [comment]
I was trying to match the repo code to what's used for the deb package

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by BMillikan** [comment]
I'm using other code that also has libUSB dependencies

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: .gitmodules [hunk 2]
```diff
@@ -4,9 +4,9 @@
 [submodule "dump978"]
 	path = dump978
 	url = https://github.com/ankurvdev/dump978
-[submodule "libusb"]
```
Comment (ankurvdev) [inline .gitmodules:7]: The fork contains a few fixes for android (from https://github.com/libusb/libusb/pull/1164)  
I'd like to keep it as is until the PR is merged. 

Testing libusb for android is a little manual and time consuming task. Happy to do a merge with main if there's something missing in the fork

Comment (ankurvdev) [reply .gitmodules:7]: There's also this 
https://github.com/libusb/libusb-cmake
which contains most of the cmake snippets from here. 
Was planning to move my fork over to this to reuse the cmake and not having to write my own

**Pull request closed without merging**